telegram/ext/dispatcher.py

```diff
@@ -162,6 +162,11 @@ def __init__(
                 stacklevel=3,
             )
 
+        if self.workers < 1:
+            self.logger.warning(
```

Comment:
On a second look, this cought my eye. In other places, we use `warnings.warn` and I have the impression, that we're not very consistent on when we use what. I think I'll just open an issue later today to discuss this in general for v14, but giving it a second thought here already might also not hurt …

Reply:
I guess we should use `logger.warning` for runtime warnings like [here](https://github.com/python-telegram-bot/python-telegram-bot/blob/9eba935b113524fea330f7fb17fcfd8db660a3c0/telegram/ext/conversationhandler.py#L481) and `warnings.warn` while initialising some class. Don't have any specific reason to so just a thought..

Reply:
Or we could use `logger.warning` everywhere except those places where we need to use `TelegramDepriciatiobWarning`.

Reply:
Did a bit of digging, see #2467. From that I'd say to use `warnings.warn` here …

Reply:
Alright 👍